spec/puppet-lint/lexer_spec.rb

```diff
@@ -140,47 +140,57 @@
     end
 
     it 'should handle a string with newline characters' do
-      @lexer.interpolate_string("foo\n    \"/bin/foo\" >>\n    /bar/baz\"", 1, 1)
+      manifest = <<-END
+    foo
+    \"/bin/foo\" >>
+    /bar/baz"
```

Comment:
The heredoc makes this much clearer, thanks!

While this may be an accurate test of the lexer, it is not very representative. How about a notify with a multi line string, for instance?

Reply:
I agree it's not representative.

I apologize for not following your request.  Do you mean doing `lexer.tokenise` for a multi-line `notice()`?

Reply:
Correct - or anything else you might have in your puppet codebase to use as a sample for testing.

Reply:
Correct - or anything else you might have in your puppet codebase to use as a sample for testing.